providers/stores/azblobDefinitionStore.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const AbstractAzBlobStore = require('./abstractAzblobStore')
+const AbstractFileStore = require('./abstractFileStore')
+const EntityCoordinates = require('../../lib/entityCoordinates')
+const { sortedUniq } = require('lodash')
+
+const responseOrError = (resolve, reject) => (error, result, response) => (error ? reject(error) : resolve(response))
+
+class AzBlobDefinitionStore extends AbstractAzBlobStore {
+  /**
+   * List all of the definitions for the given coordinates.
+   *
+   * @param {EntityCoordinates} coordinates - Accepts partial coordinates
+   * @returns A list of matching coordinates i.e. [ 'npm/npmjs/-/JSONStream/1.3.3' ]
+   */
+  async list(coordinates) {
+    const list = await super.list(coordinates, entry => {
+      const urn = entry.metadata.urn
```

Comment:
I may be misunderstanding how this is setup, but I don't think there is ever blob metadata on the definitions.

I think you can just 

`return AbstractFileStore.isInterestingCoordinates(entry.name)`